appinventor/components/tests/com/google/appinventor/components/runtime/CSVFileTest.java

```diff
@@ -0,0 +1,15 @@
+package com.google.appinventor.components.runtime;
```

Comment:
Test:
- [ ] CSVFile tests
- [ ] ChartDataBase tests?
- [ ] CoordinateData tests?

Reply:
ChartDataBase and CoordinateData to be done in #97